rollup.config.js

```diff
@@ -0,0 +1,21 @@
+import babel from 'rollup-plugin-babel';
+import pkg from './package.json';
+
+export default [
+  {
+    entry: 'src/waypoint.jsx',
+    external: [
+      ...Object.keys(pkg.dependencies),
+      ...Object.keys(pkg.peerDependencies),
+    ],
+    targets: [
+      { dest: pkg.main, format: 'cjs' },
```

Comment:
I just looked this up after reading @ljharb 's questions, and I don't see `targets` in [the Rollup docs](https://rollupjs.org/#big-list-of-options). Should this instead be `output` and `file`, rather than `targets` and `dest`?

Reply:
Looks the answer might be [yes](https://github.com/rollup/rollup/blob/6fc2088bfaf6b47b7225223398d1335f167a0aea/src/utils/deprecateOptions.js#L33-L34)

Reply:
Ooh, in that case yes that's better :-D

Reply:
Ah, good catch. I added a commit to convert these.

Reply:
I looked into updating the code I pulled this from and it looks like there is an open PR for it already! https://github.com/rollup/rollup-starter-lib/pull/2